CMakeLists.txt

```diff
@@ -22,10 +22,6 @@ if(DEFINED LF_SINGLE_THREADED)
     add_compile_definitions(LF_SINGLE_THREADED=1)
 endif()
 
-# Warnings as errors
-add_compile_options(-Werror)
```

Comment:
I see that you've added `-Werror` in the CMake config for the RTI, but what about the other modules?

Reply:
It is also added in `core/CMakeLists.txt` for when compiling the integration tests and in `test/Tests.cmake` when compiling the tests.

Reply:
:+1: